LICENSE

```diff
@@ -1,9 +1,29 @@
-Copyright (c) 2021, Center for Advanced Systems Understanding (CASUS)
+BSD 3-Clause License
 
+Copyright (c) 2021, atoMEC
```

Comment:
I would delete this line from `LICENSE` and put this into a separate `Copyright.md` file.

Copyright (c) 2021, Center for Advanced Systems Understanding
Copyright (c) 2021 (in alphabetical order), Tim Callow, Attila Cangi, Eli Kraisler 

Here, I'm just following what we did for MALA without fully understanding any legal consequences,

Reply:
I'm a little hesitant to remove this line completely from `LICENSE` because the BSD 3 template explicitly includes such a line (https://opensource.org/licenses/BSD-3-Clause). I'm a long way from an expert but maybe modifying the license in such a way could invalidate it? Happy to stand corrected if this is not true.

In any case we could include an additional `copyright` file as you suggested, and modify the first line in `LICENSE` to attribute copyright to CASUS (or our names).

Reply:
You're right, I see that the BSD-3 template includes a line for copyright. 
@DanielKotik Do you think we made a mistake with the MALA license statement?

Reply:
I am not (yet;)) a license expert, and I've also checked the templates and looked at different repositories - they all seem to include the copytright within the license. GitHub itself gives you this hint:

![grafik](https://user-images.githubusercontent.com/28047702/123397847-2e69c000-d5a3-11eb-8db0-4de67231edb7.png)

My suggestion is to stick to the license template with the filled in copyright details (to be on the safe side).

Reply:
@acangi: regarding MALA: please check if having the copyright statement in a separate file is legally ok

Reply:
I will check this. For this repo, let us go ahead with @timcallow's original `LICENSE` file, but with the replacement that list all authors as copyright holders.

Reply:
@timcallow : will you adjust the copyright as suggested by https://github.com/atomec-project/atoMEC/pull/9#discussion_r658578250 ?

Reply:
@DanielKotik have done